apps/io-wallet-user-func/src/infra/http/handlers/create-wallet-instance.ts

```diff
@@ -79,6 +112,23 @@ export const CreateWalletInstanceHandler = H.of((req: H.HttpRequest) =>
                 "NEW_WALLET_INSTANCE_CREATED",
               ),
             ),
+            RTE.chainW(() =>
+              pipe(
+                getUserEmailByFiscalCode(walletInstanceRequest.fiscalCode),
```

Comment:
if you make getUserEmailByFiscalCode an RTE maybe it becomes simpler. Also because I think you would need some configuration parameters